src/components/AddTodoForm.js

```diff
@@ -1,14 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import TodoList from "./TodoList";
 
 const AddTodoForm = () => {
+  const [TodoItem, setTodoItem] = useState("");
+  // By Bellow code we take the value of input feild and set it back to TodoItem using useState.
+  const handelChange = (event) => {
+    setTodoItem(event.target.value);
+  };
+
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    console.log(event.target.title.value);
+    event.target.title.value = "";
+  };
   return (
-    <form name="AddTodoForm">
-      <h1>Welcome to our Todo Form</h1>
+    <form name="AddTodoForm" onSubmit={handleAddTodo}>
       <div>
         <label htmlFor="TodoTitle">Todo Title</label>
-        <input type="text" id="TodoTitle" />
-        <button type="submit">Add</button>
+        <input type="text" name="title" id="TodoTitle" />
+        {/* By Bellow Code We can get Value of input feild trugh OnChange and set value. */}
+        {/* <input
+          type="text"
+          name="title"
+          id="TodoTitle"
+          onChange={handelChange}
+          value={TodoItems}
+        /> */}
+        <button type="submit">Search</button>
       </div>
+      {/* By bellow code we can get TodoItem Value and show to this place. */}
+      <p>{TodoItem}</p>
```

Comment:
Not the right place for this. Please see [Store "New Todo" in React State](https://github.com/Code-the-Dream-School/react-bald-eagle/wiki/Lesson-1.3#store-new-todo-in-react-state)